examples/rbfe_edge_list.py

```diff
@@ -59,65 +112,31 @@ def read_from_args():
         reader = csv.reader(csvfile, delimiter=",")
         next(reader)
         rows = [row for row in reader]
-        for row_idx, row in enumerate(rows):
+        for row in rows:
```

Comment:
Small orthogonal change: seemed simpler to use the same seed for each edge. Can add this back in if there's a good reason.

Reply:
Added this back in to reduce the diff.